### PR TITLE
Add nav modal working test

### PR DESCRIPTION
**Pull request by aspitz1**
## What does this PR do?
- [X] New feature (adds functionality)
- [ ] Debugging/Refactor
 
### Where should your reviewer start?
- Start in `src/components/NavMenu.js`, `NavMenu` is imported into `src/components/HeaderNavBtn`, `HeaderNavBtn` is imported into `App` and injected into the options of `Stack.Screen`
### What (if any) features are you implementing?
- Builds out the beginning stages of creating a hamburger menu
- Currently using placeholder text, and an actual hamburger emoji, I am guessing all that will change
### What (if anything) did you refactor?
- Added `options` to `Stack.Screen`
### What issues does this PR close?
- N/A
### Is there anything that you need from your teammates?
- Would one of you like to take this on and continue building it out? I am happy to continue, but I thought it would be helpful to share the functionality on this
- I am also open to a different approach
### Any other comments, questions, or concerns?
- I think it is beneficial to build our own "drawer" not only b/c npm packages can be unreliable, but also to understand the inner workings of how components are build
  - I didn't really realize a modal is probably what the packages are doing under the hood
- There are other animations the modal can use to come in and out, I just chose `fade`, I think there is `slide` too
### How Has This Been Tested?
- [X] I tested my changes in the browser
- [ ] I tested with Cypress

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aspitz1** [comment]
One other note, this doesn't have to be merged in if ya'll don't want to continue to build it out :) I just thought it may be helpful to see my approach at this point. I am happy to explore other options. I do like adding the menu to the React Navigator menu though. I think it creates a nice navigation/header space.

**Comment by aspitz1** [comment]
Here are some screen shots too. The first is the screen before the burger is pressed, the second is after the burger is pressed. Again, the burger emoji is just a place holder 👍 

Screen Shot 2022-10-16 at 7 56 57 AM
Screen Shot 2022-10-16 at 7 57 10 AM

File: App.js [hunk 1]
```diff
@@ -1,25 +1,46 @@
-import { StatusBar } from "expo-status-bar";
-import React, { Component } from 'react';
```
Comment (NickLiggett) [inline App.js:2]: Thanks for removing Component! I also noticed it was dead code.

File: src/components/HeaderNavBtn.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { View, Pressable, Text, StyleSheet } from "react-native";
+
+import NavMenu from "./NavMenu";
+
+const HeaderNavBtn = ({ modalVisible, setModalVisible }) => {
+  return (
+    <View>
+      <Pressable onPress={() => setModalVisible(true)}>
+        <Text style={styles.hamburger}>🍔</Text>
```
Comment (NickLiggett) [inline src/components/HeaderNavBtn.js:10]: Yummy...

File: src/components/HomeScreen.js [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
+  );
+};
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: "#fff",
     alignItems: "center",
-    justifyContent: "center",
+    justifyContent: "center"
```
Comment (NickLiggett) [inline src/components/HomeScreen.js:25]: Stupid comma...

File: src/components/NavMenu.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+const styles = StyleSheet.create({
+  textContainer: {
+    justifyContent: "center",
+    alignItems: "center",
+    paddingTop: 70,
+  },
+});
+
+export default NavMenu;
```
Comment (NickLiggett) [inline src/components/NavMenu.js:26]: *Sweet component dude*

**Review by NickLiggett — APPROVED** [review]

File: src/components/HomeScreen.js (hunk 3 above)
Comment (aspitz1) [reply src/components/HomeScreen.js:25]: Thank you for fixing that!

**Review by tfettig22 — APPROVED** [review]
Nice use of the modal to start our hamburger menu! Thanks Anna

**Pull request merged**